### PR TITLE
docs: guidelines for contributing documentation

### PR DESCRIPTION
**Pull request by ghuntley**
## PR Type

What kind of change does this PR introduce?

- Documentation content changes

## Other information

Opening early as a continual work-in-progress pull-request to dump my brain and seek feedback. I expect this PR to be open for quite some time.

### CODE REVIEW

File: doc/articles/contributing/guidelines/documentation.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
+# Guidelines for documentation
```
Comment (ghuntley) [inline doc/articles/contributing/guidelines/documentation.md]: 🎗 add this to the TOC

File: doc/articles/contributing/guidelines/documentation.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+- reference docs (public api reference ie class members/what's implemented/per control information)
+
+
+## building the documentation locally
+
+tba. Any advice? It should be as simple as `make documentation` or `build.ps1 documentation`
+
+## always update the table of content
+
+If new documentation is created without adding it to the table of contents then CI will fail the build. See https://github.com/unoplatform/uno/pull/1221
```
Comment (davidjohnoliver) [inline doc/articles/contributing/guidelines/documentation.md]: great idea :)

**Comment by MartinZikmund** [comment]
@ghuntley Similar here, should we close the PR for now?

**Comment by jeromelaban** [comment]
Let's close this one, @davidjohnoliver updated many of those topics.

**Pull request closed without merging**